Dictionary/Dictionary.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace Dictionary
+{
+    public static class ExercisesDictionary
+    {
+        private static void Main()
+        {
+            Exercise1();
+            Exercise2();
+            Exercise3();
+            Exercise4();
+
+            void Exercise1()
+            {
+                Console.WriteLine("Exercise 1.");
+
+                var employees = new Dictionary<int, Student>
+                 {
+                { 1, new Student(4, "John") },
+                { 2, new Student(5, "Henry") }
+                  };
+                foreach (KeyValuePair<int, Student> pair in employees)
+                {
+                    Console.WriteLine($"Student {pair.Value.Name}, estimation: {pair.Value.AverageRating}.");
+                }
+            }
+
+            void Exercise2()
+            {
+                var dictionaryEnglishFranch = new Dictionary<string, string>(10)
+                {
+                    { "book", "livre" },
+                    { "mouse", "souris" },
+                    { "dictionary", "dictionnaire" },
+                    { "notepad", "bloc-notes" },
+                    { "word", "mot" },
+                    { "time", "temps" },
+                    { "music", "musique" },
+                    { "game", "jeu" },
+                    { "flower", "fleur" },
+                    { "candles", "bougies" }
+                };
+                string targetValue = dictionaryEnglishFranch.FirstOrDefault(pair => pair.Key == "music").Value;
```

Comment:
when we say that a program accepts a value, it means it must have an external argument.
In your case, it should be in the method so that you can pass any word to the method, and it will translate into French (theoretically)